framework/src/wifi_manager/slsi_wifi_utils.c

```diff
@@ -290,6 +313,12 @@ wifi_utils_result_e wifi_utils_init(void)
 			ndbg("[WU] [ERR] Register Scan Callback(%d)\n", ret);
 			return wuret;
 		}
+
+		memset((void*)&scan_filter_result, 0, sizeof(scan_filter_result_t));
```

Comment:
Please add space between `void` and `*`.

Reply:
Fixed, thanks